.github/workflows/performance.yml

```diff
@@ -53,63 +53,63 @@ jobs:
         run: |
           set -io pipefail
           EXIT_STATUS=0
-          source scripts/check_secrets.sh
-          cd performance/SwiftUIPerformanceQuickstart/PerformanceExample
-          bundle exec pod install --repo-update
+          ./scripts/check_secrets.sh
```

Comment:
Might need to be `source scripts/check_secrets.sh`

Reply:
Yep, just reverted that change 👍